app/config.py

```diff
@@ -256,6 +256,9 @@ def AZURE_AD_AUTHORITY(self) -> str:
 
     # Service Desk
     SERVICE_DESK_URL: str = "https://mhclgdigital.atlassian.net/servicedesk/customer/portal/5"
+    # TODO update this when we create an access funding option in the service desk
```

Comment:
1344 does look like an Access specific service desk URL already? 🤔

Reply:
D'oh, forgot to remove the comment!